python/desc/twinkles/sqlite_tools.py

```diff
@@ -29,3 +31,30 @@ def create(self, columns, table, condition=None):
             query += ' ' + condition
         data = [row for row in self.conn.execute(query)]
         return pd.DataFrame(data=data, columns=columns)
+
```

Comment:
@jchiang87 I understand that, but am worried that someone will try to use this to get dataframes from the opsim database and not notice the duplicates. In principle, since the fact that these duplicates exist are in the OpSim documentation, you could say it is the user's responsibility ...